proc/variables.go

```diff
@@ -442,6 +447,13 @@ func (scope *EvalScope) packageVarAddr(name string) (*Variable, error) {
 	return nil, fmt.Errorf("could not find symbol value for %s", name)
 }
 
+func (v *Variable) toFieldNilChecked(field *dwarf.StructField, name string) (*Variable, error) {
+	if v.Addr == 0 {
```

Comment:
Thanks - I've moved the null check to `toField` now.  I don't see any callers who expected this to allow nil receivers, so I think this is okay.